### ISSUE TITLE
Program quit exceptionally after executing the function of transforms.Normalize((0.5,), (0.5,))

### ISSUE THREAD

**Issue by songyuc** [pytorch/pytorch#31420]
## 🐛 Bug

Program quit exceptionally at executing  the function of transforms.Normalize((0.5,), (0.5,))

## To Reproduce
Reproducing it needs my data.
And I want to know how can I send it to you? 

1. Loaded my data into im0 as np.ndarray [28,28]
2. Expanding the dimensions of im0 as `im1 = np.expand_dims(im0, axis=2)`
3. Getting a deep copy as `a = im1.copy()`
4. Transformed it into a torch.Tensor as
```
import torchvision.transforms as transforms
f1 = transforms.ToTensor()
im2 = f1(a)
im2 = im2.float()
im2.shape
```
5. Then did the normalizing as
```
f2 = transforms.Normalize((0.5,), (0.5,))
im3 = f2(im2)
im3.shape
```
and the program quitted exceptionally.

![image](https://user-images.githubusercontent.com/27288110/71084238-cd07aa80-21cf-11ea-913e-bc60b6592958.png)

## Expected behavior

The program quitted as,
![image](https://user-images.githubusercontent.com/27288110/71084238-cd07aa80-21cf-11ea-913e-bc60b6592958.png)

## Environment

Please copy and paste the output from our
[environment collection script](https://raw.githubusercontent.com/pytorch/pytorch/master/torch/utils/collect_env.py)
(or fill out the checklist below manually).

You can get the script and run it with:
```
wget https://raw.githubusercontent.com/pytorch/pytorch/master/torch/utils/collect_env.py
# For security purposes, please check the contents of collect_env.py before running it.
python collect_env.py
```

 - PyTorch Version (e.g., 1.0): 1.3.1
 - OS (e.g., Linux): Win10
 - How you installed PyTorch (`conda`, `pip`, source): conda
 - Build command you used (if compiling from source): by PyCharm
 - Python version: 3.7.3
 - CUDA/cuDNN version: CUDA 10.1
 - GPU models and configuration: using CPU
 - Any other relevant information:

**Comment by ptrblck** [comment]
Minimal code snippet:
```python
x = torch.randint(1, 255, (1,), dtype=torch.long)
x.div_(torch.tensor([0]))
```

`gdb` stacktrace:
```
Program received signal SIGFPE, Arithmetic exception.
0x00007fffb9406656 in void c10::function_ref<void (char**, long const*, long)>::callback_fn<void at::native::(anonymous namespace)::cpu_kernel<at::native::(anonymous namespace)::div_kernel(at::TensorIterator&)::{lambda()#1}::operator()() const::{lambda()#4}::operator()() const::{lambda(long, long)#1}>(at::TensorIterator&, at::native::(anonymous namespace)::div_kernel(at::TensorIterator&)::{lambda()#1}::operator()() const::{lambda()#4}::operator()() const::{lambda(long, long)#1})::{lambda(char**, long const*, long)#1}>(long, char**, long const*, long) ()
```

**Comment by cpuhrsch** [comment]
Likely a duplicate for the division by 0 behavior - cc @fmassa

**Comment by gen2020code** [comment]
I used the tif image to normalize and this problem also came up, and it hasn't been solved yet

**Comment by cpuhrsch** [comment]
@gen2020code @songyuc - could I ask you to open an issue over at [torchvision](https://github.com/pytorch/vision)?
